app/code/Magento/Directory/Model/ResourceModel/Country/Collection.php

```diff
@@ -237,7 +237,7 @@ public function addCountryIdFilter($countryId)
      * @param string|boolean $emptyLabel
      * @return array
      */
-    public function toOptionArray($emptyLabel = ' ')
+    public function toOptionArray($emptyLabel = false)
```

Comment:
Provided changes are backward incompatible, please, follow [Backward compatible development documentation](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/#modifying-the-default-values-of-optional-arguments-in-public-and-protected-methods)